src/DataStructures/Tensor/Structure.hpp

```diff
@@ -180,6 +197,27 @@ constexpr auto compute_collapsed_to_storage(
   }
 }
 
+/// \brief Computes a 1-1 mapping from a storage_index to its canonical
+/// tensor_index
+///
+/// \details
+/// When symmetries are present, not all unique tensor_indices can be retrieved
+/// from this map, as some tensor_indices will share the same canonical form.
+/// Otherwise, if no symmetries are present, each unique tensor_index is already
+/// in the canonical form, and one that is not shared by another tensor_index,
+/// so this would equivalently mean a 1-1 mapping from a storage_index to a
+/// tensor_index for each unique tensor_index. This means that when no
```

Comment:
I suggest removing the final "for each unique tensor_index" because I'm not sure what it means to have a 1-1 map from A to B for each A...